Azure/azure-cli
# {Graph} Rerun keyvault tests related with `az ad/role`

**Description**

As #22174 mentioned, keyvault tests have used `az ad app` commands. Since there's underlying dependency change from AAD Graph to MS Graph, recordings have to be regenerated. 

**Testing Guide**

Switch to each profile to run all profiles' tests:
- latest: `az cloud set -n AzureCloud --profile latest`
- 2018-03-01-hybrid: `az cloud set -n AzureCloud --profile 2018-03-01-hybrid`
- ...

And run tests:
`azdev test keyvault`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
